schemas/openshift/cluster-1.yml

```diff
@@ -75,8 +75,32 @@ properties:
             type: string
             enum:
             - oidc
+          name:
+            type: string
+          issuer:
+            type: string
+            format: uri
+          claims:
+            email:
+              type: array
+              items:
+                type: string
+            username:
```

Comment:
do we need a preferred username config option as well?

Reply:
`username` is the attribute for OCM `preferred_username`

Reply:
the ocm api provides an option to specify both - see `OpenIdClaims.preferred_username` on https://api.openshift.com/#/

thats why i thought it might be relevant. if it is not relevant in our context, feel free to ignore

Reply:
`username` will be translated to `preferred_username` by the [integration](https://github.com/app-sre/qontract-reconcile/pull/3029/files#diff-035de9609235cce528e8b2280f5b2d18e688e611ac923346eb60c8ff2c5f9f77R998)
I don't see `username` and `preferred_username` in the `OpenIDClaims`

![image](https://user-images.githubusercontent.com/219834/205035858-34962c5e-50a5-403e-a772-44b025fa9fb9.png)